### DISCUSSION TITLE
need the process to authenticate aws cognito for rest api

### DISCUSSION THREAD

**Discussion by RaviChandraOfficial** [awslabs/aws-sdk-rust#1105] [Q&A] [answered]
i have created rest api using rust through axum and sqlx crate. Now i have to authenticate using aws cognito, how to do it, can anyone please provide the details!

**Comment by jmklix** [comment, marked as answer]
Take a look at our cognito examples which you can find here: https://docs.aws.amazon.com/code-library/latest/ug/rust_1_cognito-identity-provider_code_examples.html
